policyengine_canada/variables/gov/provinces/qc/tax/income/credits/fa/qc_fa_payment.py

```diff
@@ -0,0 +1,60 @@
+from policyengine_canada.model_api import *
+
+
+class qc_fa_payment(Variable):
+    value_type = float
+    entity = Household
+    label = "Quebec family allowance payment amount"
+    reference = "https://www.legisquebec.gouv.qc.ca/en/document/cs/I-3?langCont=en#se:1029_8_61_18"
+    definition_period = YEAR
+    defined_for = "qc_fa_eligibility"
+
+    def formula(household, period, parameters):
+        p = parameters(period).gov.provinces.qc.tax.income.credits.fa
+
+        eligibility = household("qc_fa_eligibility", period)
```

Comment:
don't need since it's in `defined_for`

Reply:
Thank you for letting me know. I was wondering if we need to have `qc_fa_eligibility` as an input in test files?

Reply:
yes you do